ibis/expr/operations/core.py

```diff
@@ -44,6 +44,11 @@ def equals(self, other):
             )
         return self.__cached_equals__(other)
 
+    def _op(self):
```

Comment:
I think the following should work:

```py
@deprecated(version="4.0", instead="remove intermediate .op() calls")
def op(self):
    return self
```

Reply:
Also we need a test case for it.

Reply:
> I think the following should work:

Of course.  I don't know what I was thinking, thanks for catching it.

Reply:
I don't think a test is necessary here. The current CI failures indicate the code is exercised, and we aren't going to keep this around forever. Enshrining the behavior with a test is undesirable IMO.